Userland/Applications/Magnifier/main.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2021, Valtteri Koskivuori <vkoskiv@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "MagnifierWidget.h"
+#include <LibGUI/Application.h>
+#include <LibGUI/Icon.h>
+#include <LibGUI/Menu.h>
+#include <LibGUI/Menubar.h>
+#include <LibGUI/Window.h>
+#include <unistd.h>
+
+int main(int argc, char** argv)
+{
+    if (pledge("stdio cpath rpath recvfd sendfd unix fattr", nullptr) < 0) {
+        perror("pledge");
+        return 1;
+    }
+
+    auto app = GUI::Application::construct(argc, argv);
+
+    if (pledge("stdio cpath rpath recvfd sendfd", nullptr) < 0) {
+        perror("pledge");
+        return 1;
+    }
+
+    // Sneaky!
+    // FIXME: Doesn't have a 32x32 icon yet, need to make one!
+    auto app_icon = GUI::Icon::default_icon("find");
+
+    auto window = GUI::Window::construct();
+    window->set_title("Magnifier");
+    window->resize(208, 208);
```

Comment:
Magic numbers... probably 200 + whatever the 8px are? Would be nice to (`constexpr`) calculate this instead with some expressive names.